flatpak/dev.neovide.Neovide.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.neovide.Neovide</id>
+  <name>Neovide</name>
+  <summary>Advanced graphical interface for Neovim</summary>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <developer id="dev.neovide">
+    <name>Kaylee Simmons</name>
+  </developer>
+  <description>
+    <p>Fully-featured Neovim GUI with some visual upgrades:</p>
+    <ul>
+      <li>Animated cursor</li>
+      <li>Smooth scrolling</li>
+      <li>Animated windows</li>
+      <li>Blurred floating windows</li>
+      <li>Emoji support</li>
+    </ul>
+  </description>
+  <categories>
+    <category>Utility</category>
+    <category>TextEditor</category>
```

Comment:
I think we could add `Development` here. Kate for example includes that category.